projects/instant-ui-library/src/lib/button/base-button/base-button.component.scss

```diff
@@ -0,0 +1,60 @@
+@import "../../styles/global.styles";
+@import "../../styles/variables/constants";
+
+$transition-out: ease-in 0.1s;
+$transition-in: ease-out 0.3s;
+$box-shadow-color: none;
+@mixin boxShadow($color) {
+  box-shadow: 0px 2px 8px $color;
+  &:hover,
+  &:focus {
+    box-shadow: 0px 8px 15px $color;
+  }
+}
+@mixin createButton($shadow-color, $text-color) {
+  background-color: $shadow-color;
+  color: $text-color;
+  @include boxShadow($shadow-color);
+  &:disabled {
+    cursor: not-allowed;
+    background-color: $color--disabled3;
+    box-shadow: none;
+    transition: none;
+  }
+}
+.btn {
+  font-size: 12px;
+  border-radius: 6px;
+  padding: $button--padding;
```

Comment:
If we aren't going to be creating variables for the other properties like `border-radius` we can probably remove this variable.